bio/bowtie2/build/environment.yaml

```
@@ -0,0 +1,7 @@
+channels:
+  - bioconda
+  - conda-forge
+  - defaults
+dependencies:
+  - bowtie2 ==2.4.1
```

Maybe unpin the patch version, unless this particular patch version is needed (e.g. for a version-specific index or something?)?
```suggestion
  - bowtie2 =2.4
```

---

I'd be happy to do so, but the bowtie2 align wrapper pins this particular version. In any case, I suggest that these two wrappers should be consistent.

---

Jep, that sounds perfectly reasonable. So lets keep it pinned as is and these should then be updated in sync. Maybe we can add a comment here (and correspondingly in the `align` wrapper's `environment.yaml`), noting this -- thus whenever anyone does an update, they are reminded to do it in sync:
```suggestion
  # if you update this version, please also update the version at bio/bowtie2/align/environment.yaml
  - bowtie2 ==2.4.1
```

---

Done in 8fd850d.